src/sentinel.c

```diff
@@ -49,87 +49,145 @@ extern SSL_CTX *redis_tls_ctx;
     char *ip;               /* Always a resolved address */
     int port;
 } sentinelAddr;
 
 /* A Sentinel Redis Instance object is monitoring. */
+/* 实例是一个主服务器 */
 #define SRI_MASTER  (1<<0)
+/* 实例是一个从服务器 */
 #define SRI_SLAVE   (1<<1)
+/* 实例是一个 Sentinel */
 #define SRI_SENTINEL (1<<2)
+/* 实例当前状态为: SDOWN */
 #define SRI_S_DOWN (1<<3)   /* Subjectively down (no quorum). */
+/* 实例当前状态为：ODOWN */
 #define SRI_O_DOWN (1<<4)   /* Objectively down (confirmed by others). */
+/* Sentinel 认为主服务器已下线 */
 #define SRI_MASTER_DOWN (1<<5) /* A Sentinel with this flag set thinks that
                                    its master is down. */
+/* 正在对主服务器进行故障迁移 */
 #define SRI_FAILOVER_IN_PROGRESS (1<<6) /* Failover is in progress for
                                            this master. */
+/* 实例被选中为主服务器（当前仍是从服务器）*/
 #define SRI_PROMOTED (1<<7)            /* Slave selected for promotion. */
+/* 向服务器发送 SLAVEOF 命令，让其转向复制新主服务器 */
 #define SRI_RECONF_SENT (1<<8)     /* SLAVEOF <newmaster> sent. */
+/* 从服务器正在和主服务器进行同步 */
 #define SRI_RECONF_INPROG (1<<9)   /* Slave synchronization in progress. */
+/* 从服务器与新服务器同步完毕，开始复制新主服务器 */
 #define SRI_RECONF_DONE (1<<10)     /* Slave synchronized with new master. */
+/* 主服务器强制执行故障迁移操作 */
 #define SRI_FORCE_FAILOVER (1<<11)  /* Force failover with master up. */
+/* 已对返回 -BUSY 的服务器发送 SCRIPT KILL 命令 */
 #define SRI_SCRIPT_KILL_SENT (1<<12) /* SCRIPT KILL already sent on -BUSY */
+/* 主服务器正在重启 */
 #define SRI_MASTER_REBOOT  (1<<13)   /* Master was detected as rebooting */
 
+/* 时间常量，均以毫秒为单位 */
 /* Note: times are in milliseconds. */
 #define SENTINEL_PING_PERIOD 1000
 
+/* 发送 INFO 命令的间隔 */
 static mstime_t sentinel_info_period = 10000;
+/* 发送 PING 命令的间隔 */
 static mstime_t sentinel_ping_period = SENTINEL_PING_PERIOD;
+/* 发送 ASK 命令的间隔 */
 static mstime_t sentinel_ask_period = 1000;
+/* 发送 PUBLISH 命令的间隔 */
 static mstime_t sentinel_publish_period = 2000;
+/* 默认判断服务器已下线的时间 */
 static mstime_t sentinel_default_down_after = 30000;
+/* 默认 TILT 触发时间 */
 static mstime_t sentinel_tilt_trigger = 2000;
+/* 默认 TILT 环境时长 （需要多久才能退出 TITL 模式）*/
 static mstime_t sentinel_tilt_period = SENTINEL_PING_PERIOD * 30;
+/* 默认的从服务器配置修改超时时长 */
 static mstime_t sentinel_slave_reconf_timeout = 10000;
+/* 默认最少重连间隔 */
 static mstime_t sentinel_min_link_reconnect_period = 15000;
+/* 默认选举超时时长 */
 static mstime_t sentinel_election_timeout = 10000;
+/* 脚本最大执行时长 */
 static mstime_t sentinel_script_max_runtime = 60000;  /* 60 seconds max exec time. */
+/* 脚本重试前的延迟时间 */
 static mstime_t sentinel_script_retry_delay = 30000;  /* 30 seconds between retries. */
+/* 默认故障迁移执行时长 */
 static mstime_t sentinel_default_failover_timeout = 60*3*1000;
 
+/* 默认信息频道 */
 #define SENTINEL_HELLO_CHANNEL "__sentinel__:hello"
+/* 默认的从服务器优先级 */
 #define SENTINEL_DEFAULT_SLAVE_PRIORITY 100
+/* 默认同时对新服务器进行复制的从服务器数量 */
 #define SENTINEL_DEFAULT_PARALLEL_SYNCS 1
+/* 默认最大积压命令数量 */
 #define SENTINEL_MAX_PENDING_COMMANDS 100
-
+/* 启动故障迁移最大延迟时间 */
```

Comment:
```suggestion
/* 启动故障转移最大延迟时间 */
```
我们要不统一一下叫“故障转移”？百科也都是这么命名。